api/conf/settings/rest_framework.py

```diff
@@ -57,3 +57,5 @@
         **{k: None for k, _ in REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"].items()}
     )
     del REST_FRAMEWORK["DEFAULT_THROTTLE_CLASSES"]
+
```

Comment:
Would be nice to have a comment on why we set this and why set it to 3.

Reply:
We're really just setting a default, and 3 is a production implementation detail. I think this should actually default to 0 instead and then we can set the actual value in our live environments (and document the number there).

Reply:
I've updated it to default to 0 and added a comment linking to the docs and mentioning that we set a specific value in live environments.